utils/io/content/contentwriter.go

```diff
@@ -140,10 +163,19 @@ func (rw *ContentWriter) run() {
 }
 
 // Finish writing to the file.
+// To avoid creating an empty file whenever 'Write' never been called, we start the worker so an empty array will be created.
```

Comment:
Bad grammar. I couldn't understand this sentence.

Reply:
Removed.